README.md

```diff
@@ -1,7 +1,27 @@
 # DSA Cleveland Evictions Tracker
 The DSA Cleveland evictions tracker is a java command line tool designed to parse evictions data.
 
-## Getting Started
+## Windows Instructions
```

Comment:
The Windows instructions you provide are usage instructions. The linux instructions that are pushed below are build instructions for contributing to the project. Can you change the headings to distinguish between those two?

Reply:
b8aadb5 Retitled Windows section headers to match Linux

Reply:
bumping for review @JKobyP